composer.json

```diff
@@ -27,7 +27,7 @@
 	"minimum-stability": "stable",
 	"config": {
 		"platform": {
-			"php": "5.6"
+			"php": "7.2"
```

Comment:
hmmm myslim ze mam 7.1 🙈 😅 nesnizime to o fous? 😇

Reply:
PHP podporuje datové typy u class properties až od 7.4. Ale necháme teda 7.1.